### PR TITLE
[feature/heat-index-worksheets]

### PR DESCRIPTION
**Pull request by 1Copenut**
### TODO: Verify the Word doc is accessible on Windows machine when approved

### CODE REVIEW

File: worksheets/Accessibility Evaluation Checklist.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Accessibility Evaluation Checklist
+
+## What this document does
+
+This document provides guidance to assess, report, and quantify the accessibility of the products Ad Hoc builds.
+
+## Who it is for
+
+User interface and experience designers, researchers, and front-end engineers are the primary audience. However, this is not an exhaustive list. Anyone with access to a web browser or mobile device can be an accessibility tester. This document and templates can be used to help you identify **what** to test, and **how** to evaluate your website or application.
```
Comment (dannychapman) [inline worksheets/Accessibility Evaluation Checklist.md]: 'user experience designers'

File: worksheets/Accessibility Evaluation Checklist.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Screen reader testing is mostly a manual process. Ideally, pages under test should be free of errors identified by axe-core or other automated scanner before manual testing begins. For applications, this means all pages should be evaluated. For larger static sites, pick 10-15 highly trafficked pages, or run the [axe-e2e script](https://github.com/adhocteam/a11y-snippets/tree/master/test-scripts/axe-e2e) again.
+
+* [Test using JAWS + IE11 on Win7/10](https://webaim.org/articles/jaws/)
+* [Test using NVDA + Firefox on Win7/10 - VIDEO](https://www.youtube.com/watch?v=Vx1vSd5uYS8)
+* [Test using VoiceOver + Safari on OSX](https://webaim.org/articles/voiceover/)
+
+### Step 3. Audit keyboard navigation - maximal importance
+
+Keyboard testing is mostly a manual process. All pages under test should be free of errors identified by axe-core or other automated scanner first, then manual testing and ideally keyboard end-to-end testing should follow.
```
Comment (dannychapman) [inline worksheets/Accessibility Evaluation Checklist.md]: or other automated scanners

File: worksheets/Accessibility Evaluation Checklist.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+### Step 1. Tally automated errors - high importance
+
+Automated scan errors are the first thing we recommend logging for fixes. These errors can cause issues for screen reader, keyboard, mobile, and low-vision users. Issues include improperly formed markup, misused ARIA attributes, color contrast, and missing landmark structures.
+
+* [Run automated scans](https://www.youtube.com/watch?v=FW1giWW5M9I) against all rich application pages or states. The Axe-core plugin is available for [Chrome](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/). This adds a tab to developer tools and provides on-demand accessibility scanning for single pages during development.
+* [Run the axe-end to end script with CSV and console output](https://github.com/adhocteam/a11y-snippets/tree/master/test-scripts/axe-e2e) for automated scans of high page count static sites. Errors can be outputted to CSV for easy spreadsheet import.
+
+### Step 2. Audit with screen readers - maximal importance
```
Comment (dannychapman) [inline worksheets/Accessibility Evaluation Checklist.md]: Maximum

File: worksheets/Accessibility Evaluation Checklist.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+Screen reader testing is mostly a manual process. Ideally, pages under test should be free of errors identified by axe-core or other automated scanner before manual testing begins. For applications, this means all pages should be evaluated. For larger static sites, pick 10-15 highly trafficked pages, or run the [axe-e2e script](https://github.com/adhocteam/a11y-snippets/tree/master/test-scripts/axe-e2e) again.
+
+* [Test using JAWS + IE11 on Win7/10](https://webaim.org/articles/jaws/)
+* [Test using NVDA + Firefox on Win7/10 - VIDEO](https://www.youtube.com/watch?v=Vx1vSd5uYS8)
+* [Test using VoiceOver + Safari on OSX](https://webaim.org/articles/voiceover/)
+
+### Step 3. Audit keyboard navigation - maximal importance
```
Comment (dannychapman) [inline worksheets/Accessibility Evaluation Checklist.md]: maximum

File: worksheets/Accessibility Manual Testing Criteria.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+# Manual a11y Testing Pass/Fail Criteria
+
+Manual accessibility testing often comes down to subjective reasoning. If something doesn’t feel right, it probably isn’t, and should be logged as an error. Your answers will help determine the accessibility heat index score for the application or site being tested.
+
+## Screen readers - 5 or more errors
```
Comment (dannychapman) [inline worksheets/Accessibility Manual Testing Criteria.md]: consider writing out as five

**Comment by dannychapman** [comment]
It would be great to add some guidelines as to what a good achievable score is, both to the readme and the spreadsheets.

File: worksheets/Accessibility Manual Testing Criteria.md (hunk 5 above)
Comment (dannychapman) [reply worksheets/Accessibility Manual Testing Criteria.md]: I realize there is pass/fail criteria, but setting some baseline ideas as to what a typical starting score might be would be helpful.

**Comment by dannychapman** [comment]
@1Copenut Made some minor line comments but otherwise this looks good.

**Review by dannychapman — CHANGES_REQUESTED** [review]
See comments, otherwise looks good.

**Comment by 1Copenut** [comment]
@dannychapman All changes are in. I added a set of scores to the README and worksheets, that are goals to work toward. The manual errors tend to magnify scores greatly, so the algorithm may need tweaking, but it's a good first step.

**Review by dannychapman — APPROVED** [review]

**Pull request merged**